client/rest/src/plugins/rosetta/nem/OperationParser.js

```diff
@@ -192,7 +192,7 @@ export class OperationParser {
 	 */
 	async parseTransactionAsRosettaTransaction(transaction, metadata) {
 		const { operations } = await this.parseTransaction(transaction, metadata);
```

Comment:
not part of this PR, but the ``parseTransaction`` does not take ``metadata`` parameter.